wandb/sdk/launch/utils.py

```diff
@@ -163,6 +165,19 @@ def parse_wandb_uri(uri: str) -> Tuple[str, str, str]:
     return entity, project, name
 
 
+def is_bare_wandb_uri(uri: str) -> bool:
+    """Checks if the uri is of the format /entity/project/runs/run_name"""
```

Comment:
when do we expect this format? especially with the `runs` in the middle which would reasonably only appear if you're pasting from an actual url and cutting off the front

Reply:
Unfortunately some local installations seem to send the uri in the spec in this format. Filed a ticket for it: https://wandb.atlassian.net/browse/WB-7594